app/views/admin/admins/index.html.haml

```diff
@@ -1,21 +1,5 @@
-%h1 Admins Management
+%h1 Admin Management
 
 %p= link_to "Add new admin", new_admin_admin_path, class: "btn"
 
-%table
-  %col{style: 'width: 50%'}
-  %thead
-    %tr
-      %th= sortable_heading_for(:email)
-      %th.actions Actions
-  %tbody
-    - @users.each do |user|
-      %tr
-        %td= user.email
-        %td.actions
-          - if policy(user).edit?
-            = link_to "Edit", edit_admin_admin_path(user), class: "btn btn-small"
-          - if policy(user).destroy?
-            = link_to "Delete", admin_admin_path(user), method: :delete, data: { confirm: "Are you sure you want to delete #{user}?" }, class: "btn btn-small"
-
-= paginate @users
+= render partial: "user_results", locals: {delete_url: -> (user) { admin_admin_path(user) }, edit_url: -> (user) { edit_admin_admin_path(user) } }
```

Comment:
@osiro - the partial is used so that for each type of user a different path can be provided.

EG: For members it's `admin_members_path` and for admins it's `admin_admins_path`